.azure-pipelines/pipelines.yml

```diff
@@ -50,6 +50,19 @@ jobs:
         parameters:
           ciTarget: bazel.release
 
+  - job: release_arm64
```

Comment:
qq: I think I missed all of the dev here, but where is this going to push images? Or that's not done yet and will be done next?

Reply:
That's not done, also not all test are passing. So this is only enabled in post submit so we know whether it build. This is only part of #1861.

Reply:
OK sounds good. Excited for this!